src/index.d.ts

```diff
@@ -52,6 +52,8 @@ export interface MarkdownParser {
 export interface ASTNode {
   type: string;
   sourceType: string; // original source token name
+  sourceInfo: string;
+  sourceMeta: string | null;
```

Comment:
According to [@types/markdown-it](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/markdown-it/lib/token.d.ts#L69), `meta` is of type `any`.